File: java Assignment/Assignment4/src/CurvedBanner.java
package edu.met.banking;
public class CurvedBanner extends Banner{
	private double radius;

	public CurvedBanner(double w,double h,double r){
		super(w,h);
		radius = r;
	}
	public double bannerArea(){
		double area=super.bannerArea()-0.86*radius*radius;
		return area;
	}
}

File: Kapil RNO 39/DotNet/Assignment 3/DemoApp/Banner/CurvedBanner.cs
namespace banner;

class CurvedBanner : Banner
{
    public float radius { get; set;}
     
     public CurvedBanner(float width, float height, float radius)
     {
        base.width = width;
        base.height = height;
        this.radius = radius;
     }
    public override double Area()
    {
        return base.Area() - 0.86 * radius * radius;
    }

}